services/movie_session.py

```diff
@@ -42,3 +44,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider adding exception handling for `MovieSession.DoesNotExist` to handle cases where the movie session with the given ID does not exist.